stdlib/compression/_common/_streams.pyi

```diff
@@ -0,0 +1,25 @@
+from _typeshed import WriteableBuffer
```

Comment:
```suggestion
from _typeshed import Incomplete, WriteableBuffer
```

See below.

Reply:
I just copied that file from `_compression.pyi` without any changes, should I make the requested changes on `_compression.pyi` as well?

Reply:
That would be great, thanks!

Reply:
That should be done now!

Do you have any thoughts on the code duplication between the two? Like a way to remove the duplication, or if I should add a comment on both files telling that the other should probably be updated at the same time.

Reply:
Maybe it would make sense to add a comment to the top of `_compression.pyi` to copy all changes to `compression.*`. I don't think the reverse direction is as important (and often the changes won't apply to older Python versions anyway.) Maybe add a comment to `DecompressReader.decomp_factory` in case someone actually adds a proper return type at some point.

Reply:
I have added the two comments, hopefully the wording is ok